### ISSUE TITLE
Add keras.ops.cov

### ISSUE THREAD

**Issue by Neilblaze** [keras-team/keras#23493]
## Summary

`keras.ops.corrcoef` is available, but the covariance matrix it normalizes is not. NumPy, JAX and PyTorch all ship `cov` alongside `corrcoef`.

## Motivation

Covariance is the base quantity and `corrcoef` is derived from it, so having the derived one without the base is an odd place to stop. It also comes up on its own in whitening, Mahalanobis distance, Gaussian log likelihoods and PCA, none of which want the correlation form.

Without it the centered matmul has to be written by hand, and then repeated per backend if the code needs to stay backend agnostic.

## Proposed API

```python
keras.ops.cov(x)
```

Matching `np.cov` for a 2D input of shape `(N, D)`, where N is the number of variables and D the number of observations, normalized by `D - 1`. A 1D input, or a single variable, yields a scalar.

---

Yes, I would like to send a PR.
